pytest_html/plugin.py

```diff
@@ -211,15 +211,16 @@ def _generate_report(self, session):
         numtests = self.passed + self.failed + self.xpassed + self.xfailed
         generated = datetime.datetime.now()
 
-        style_css = pkg_resources.resource_string(
+        self.style_css = pkg_resources.resource_string(
             __name__, os.path.join('resources', 'style.css'))
         if PY3:
-            style_css = style_css.decode('utf-8')
+            self.style_css = self.style_css.decode('utf-8')
 
         head = html.head(
             html.meta(charset='utf-8'),
             html.title('Test Report'),
-            html.style(raw(style_css)))
+            html.link(href='style.css', rel='stylesheet', type='text/css'),
+            html.style(raw(self.style_css)))
```

Comment:
We could do the same with JavaScript, but by default Jenkins disallows all JavaScript regardless of it being inline or in an external file. Consider splitting the JavaScript into it's own file as outside of the scope of #29, but feel free to work on it once the main work is done.